tests/test_hashfields.py

```diff
@@ -1,8 +1,14 @@
+import os
 import subprocess
 
 import pytest
 
 
+def normalize_output(output: str):
+    """Normalizes string output across OSes"""
+    return output.replace("\r\r", "\r").strip()
```

Comment:
Should this be `\r\n`?

Reply:
I see the tests did pass on Windows, so I guess this is working? But I thought `\r\n` was the end-of-line terminator for Windows and haven't seen `\r\r` before.

Reply:
Yeah maybe I should have a comment, but it was only for tests so I didn't care that much..

This is coming from output captured from the `stdout` and `stderr`, there is an additional `\r` in front of the newline in Windows. You should be able to try this on your local Windows machine to see it (in a Git Bash terminal):

```bash
# activate virtual env
python -m venv .venv
source .venv/Scripts/activate

# install this lib
pip install -e .[test]
```

Now run the tests in VS Code with a breakpoint set on one of the calls to this `normalize_output()` from `capfd` -- you'll see the extra `\r` in there! Arrggg 🤯

Reply:
Ah I see. Thanks!